tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,658 @@
+			}
+		}
+
+		csiNamespace = GetAndExpectStringEnvVar(envCSINamespace)
+		vcAddress = e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
+		dataCenter = GetAndExpectStringEnvVar(datacenter)
+		computeCluster = GetAndExpectStringEnvVar(envComputeClusterName)
+		datastoreName = GetAndExpectStringEnvVar(envNfsDatastoreName)
+		datastoreIP = GetAndExpectStringEnvVar(envNfsDatastoreIP)
+		kubeconfig1 = GetAndExpectStringEnvVar("KUBECONFIG1")
+
+		ginkgo.By("Getting User and Supervisor-Id for both the supervisors")
+		for i := 0; i <= numberOfSvc-1; i++ {
+			vsphereCfg := getSvcConfigSecretData(clients[i], ctx, csiNamespace)
+			supervisorIds = append(supervisorIds, vsphereCfg.Global.SupervisorID)
+			// Getting service account user without domain name after spilittin it by @
+			wcpServiceAccUsers = append(wcpServiceAccUsers, strings.Split(string(vsphereCfg.Global.User), "@")[0])
+		}
+
+		sshClientConfig = &ssh.ClientConfig{
+			User: "root",
+			Auth: []ssh.AuthMethod{
+				ssh.Password(nimbusVcPwd),
+			},
+			HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+		}
+
+	})
+
+	ginkgo.AfterEach(func() {
+		framework.Logf("log-from-after-each")
+		for i := 0; i <= numberOfSvc-1; i++ {
+			setResourceQuota(clients[i], namespaces[i], defaultrqLimit)
+		}
+	})
+
+	/* TESTCASE-1
+
+	Create multiple SVCs and create workloads on it
+
+	Steps:
+		1. Verify the SVC account is different for each cluster
+		2. Create sts on each of the clusters and verify that it is successful
+		3. Delete all the sts and pvcs which belong to them
+	*/
+
+	ginkgo.It("[csi-multi-svc] Workload creation on each of the clusters", ginkgo.Label(p0, multiSvc, stable), func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var err error
+		for n := 0; n < numberOfSvc; n++ {
+			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyNames[n])
+			scParametersList[n][scParamStoragePolicyID] = profileID
+
+			client := clients[n]
+			namespace := namespaces[n]
+			if n == 1 {
+				os.Setenv(kubeconfigEnvVar, kubeconfig1)
+				framework.TestContext.KubeConfig = kubeconfig1
+				client, err = createKubernetesClientFromConfig(kubeconfig1)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred(),
+					fmt.Sprintf("Error creating k8s client with %v: %v", kubeconfig1, err))
+			}
+			// create resource quota
+			createResourceQuota(client, namespace, rqLimit, defaultNginxStorageClassName)
+
+			scSpec := getVSphereStorageClassSpec(defaultNginxStorageClassName, scParametersList[n], nil, "", "", false)
+			sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+
+			ginkgo.By("Create StatefulSet and verify pv affinity and pod affinity details")
```

Comment:
fix this comment, it is not a topology setup

Reply:
fixed